cmd/projects/list_test.go

```diff
@@ -115,8 +116,43 @@ func TestList(t *testing.T) {
 			"--base-uri", "invalid",
 		}
 
-		_, err := cmd.CallCmd(t, clients, args)
+		_, err := cmd.CallCmd(t, clients, &analytics.NoopClient{}, args)
 
 		assert.EqualError(t, err, "base-uri is invalid"+errorHelp)
 	})
+
+	t.Run("will track analytics for 'CLI Command Run' event", func(t *testing.T) {
```

Comment:
Can you remove the quotes for these tests? It messes up the test name when running with `-v`.